grafana/data_source_amixr_action.go

```diff
@@ -0,0 +1,50 @@
+package grafana
+
+import (
+	"fmt"
+	amixrAPI "github.com/grafana/amixr-api-go-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceAmixrAction() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+* [HTTP API](https://grafana.com/docs/grafana-cloud/oncall/oncall-api-reference/outgoing_webhooks/)
```

Comment:
I believe this is going to the wrong page, maybe this is the right URL?
```suggestion
* [HTTP API](https://grafana.com/docs/grafana-cloud/oncall/oncall-api-reference/)
```

Reply:
The link is correct, action and outgoing webhook are the same